spec/views/layouts/list_grid.html.haml_spec.rb

```diff
@@ -10,4 +10,34 @@
       render
     end
   end
+
+  it "has valid links for row clicks" do
+    options = {:grid_id => 'list_grid', :grid_name => 'gtl_list_grid',
+
+                :grid_hash => {:head => [{:is_narrow => true}, {:is_narrow => true},
+                                         {:text => 'Name', :sort => 'str', :col_idx => 0, :align => 'left'},
+                                         {:text => 'Description', :sort => 'str', :col_idx => 0, :align => 'left'},
+                                         {:text => 'Type', :sort => 'str', :col_idx => 0, :align => 'left'},
+                                         {:text => 'Display in Catalog', :sort => 'str', :col_idx => 0, :align => 'left'},
+                                         {:text => 'Catalog', :sort => 'str', :col_idx => 0, :align => 'left'},
+                                         {:text => 'Created On', :sort => 'str', :col_idx => 0, :align => 'left'}],
+                               :rows => [{:id => '10r96',
+                                          :cells => [{:is_checkbox => true},
+                                                     {:tile => 'View this item', :image => '/pictures/10r33.png'},
+                                                     {:text => 'abcd'}, {:text => 'abcd'}, {:text => 'Item'},
+                                                     {:text => 'Yes'}, {:text => ''},
+                                                     {:text => '01/10/16 22:30:00 UTC'}]
+                                         },
+                                         {:id => '10r28',
+                                          :cells => [{:is_checkbox => true},
+                                                     {:tile => 'View this item', :image => '/pictures/10r33.png'},
+                                                     {:text => 'efgh'}, {:text => 'efgh'}, {:text => 'Item'},
+                                                     {:text => 'Yes'}, {:text => ''},
+                                                     {:text => '01/08/16 20:30:00 UTC'}]
+                                         }]},
+                :button_div => 'center_tb', :action_url => 'explorer'}
```

Comment:
why is this test checking the navigation to explorer? isn't the issue when navigating from a list view to a single entity? (not in the explorer)